CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ### Added
 
+- Self-hosted etcd will mount a hostPath volume directory defined as
+  value of annotation `selfhosted-etcd.alpha.coreos.com/checkpointdir`.
```

Comment:
@aaronlevy @diegs 
Disregard this.
etcd operator is not gonna add any new fields in next release.